clients/runner/src/runner.rs

```diff
@@ -568,10 +562,6 @@ impl RunnerExt for Runner {
 		Runner::try_load_downloaded_binary(self, release)
 	}
 
-	async fn get_request_bytes_wrapper(&self, url: String) -> Result<Bytes, Error> {
```

Comment:
same case with https://github.com/pendulum-chain/spacewalk/pull/513/files#r1577590058

Reply:
Also I think it is fine to remove.